packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -745,31 +750,34 @@ class ConstrainedBox extends SingleChildRenderObjectWidget {
   }
 }
 
-/// Sizes itself to a fraction of the total available space.
+/// An overflow box that sizes its child to a fraction of the total available space.
```

Comment:
I lifted the name from the widget's renderer, RenderFractionallySizedOverflowBox. I thought we used the name to identify widgets that didn't pass the constraints they were given to their child. 

Even if all of that is true, it's not likely that our dear reader will know as much, so I took the whole qualifying expression out.